src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -156,8 +156,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     // - Style our UI elements based on the values in our _settings, and set up
     //   other control-specific settings. This method will be called whenever
     //   the settings are reloaded.
-    //   * Sets up the background of the control with the provided BG color,
-    //      acrylic or not, and if acrylic, then uses the opacity from _settings.
+	//   * Prioritizes the acrylic background if chosen, respecting acrylicOpacity

```

Comment:
These comment lines look indented a little off

Reply:
Oops, missed these on the last commit.  Fixed.